packages/schematics/angular/utility/project.ts

```diff
@@ -35,6 +43,23 @@ export function getProject<TProjectType extends ProjectType = ProjectType.Applic
   return workspace.projects[projectName] as WorkspaceProject<TProjectType>;
 }
 
+export function getProjectConfig(host: Tree, path: string) {
+  const configName = path.split('/').pop();
```

Comment:
you can use `path.basename(path)` for this.

Reply:
Thanks for noting :+1: